### PR TITLE
Reid master

### PR DESCRIPTION
**Pull request by theJenix**
Patch to reid's fix for throttling toast messages

### CODE REVIEW

File: SoundStream/src/com/thelastcrusade/soundstream/util/Toaster.java [hunk 1]
```diff
@@ -29,18 +33,57 @@
...
+    public static synchronized void toastWorker(Context context, final String s) {
+        if (isShown(s)) {
+            return;
+        }
+        lastShown.put(s, System.currentTimeMillis());
         Context ac = context.getApplicationContext();
+        //this enables us to send more toasts, in some time
+        setResetEvent(ac, s);
+        
+        //toast the toast!
```
Comment (dgreenhalgh) [inline SoundStream/src/com/thelastcrusade/soundstream/util/Toaster.java:58]: How does this affect our approach to #269?

Comment (theJenix) [inline SoundStream/src/com/thelastcrusade/soundstream/util/Toaster.java:58]: I think it resolves it, right?

**Comment by dgreenhalgh** [comment]
@twrobel3, wanna take a look at this so we can get it pulled in?

**Comment by taywrobel** [comment]
Looks good to me, approved.

**Pull request merged**